docs/customizing_your_tooltip.md

```diff
@@ -0,0 +1,55 @@
+# Customizing Your Tooltip
+If you are looking at this page, you should already have created a default tooltip. If you haven't, please read [Creating Your Tooltip](creating_your_tooltip.md).
+
+You can customize the look and the content of your tooltip by passing in an optional `options` parameter to `vl.tooltip()` or `vg.tooltip()`. For example:
+
+```js
+// in vg.embed callback
+vl.tooltip(result.view, vlSpec, options);
+```
+or
+```js
+// in vg.parse.spec callback
+vg.tooltip(view, options);
+```
+
+The complete structure of `options` looks like this:
+
+```js
+var options =
+{
+    showAllFields: true | false,
+    fields: [
+      {
+         field: 'field1',
+         title: 'Field One',
+         formatType: 'time' | 'number' | 'string', 			
+         format: string-specifier,
+         aggregate: operation,
+      },
+      ...
+    }],
+    colorTheme: 'light' | 'dark'
+};
+```
+
+Here are the list of options:  (All of them are optional)
+
+| Option | Description |
+| :------| :-----------|
+| `showAllFields` | A boolean that specifies whether tooltip should show all fields or only fields in the `options.fields[]` array. If `showAllFields` is `true`, tooltip shows all fields bind to the mark being hovered. The order of the fields in the tooltip is the order they are bind to the mark item. If `showAllFields` is `false`, tooltip only shows fields specified in the `fields[]` array. The order of the fields in the tooltip is the same as the order specified in the `fields[]` array. `showAllFields` defaults to `true`. |
```

Comment:
> `showAllFields` defaults to `true`. 

See Vega-Lite docs for how do we display default value, we should use consistent style.